leftwm-core/src/handlers/command_handler.rs

```diff
@@ -567,6 +568,28 @@ fn focus_window_change(
     Some(handle_focus(state, handle))
 }
 
+fn focus_window_top(state: &mut State, toggle: bool) -> Option<bool> {
+    let tag = state.focus_manager.tag(0)?;
+    let cur = state.focus_manager.window(&state.windows).map(|w| w.handle);
+    let prev = state.focus_manager.tags_last_window.get(&tag).copied();
+    let next = state
+        .windows
+        .iter()
+        .find(|x| x.tags.contains(&tag) && !x.is_unmanaged())
+        .map(|w| w.handle);
+
```

Comment:
You may want to do && !x.floating(), otherwise it will move focus to the top floating window if there are any (unless that is desired?).

Reply:
Fixed that, as that is definitely not the desired effect of this feature imo